exporters/trace/gcp_exporter/recordable.h

```diff
@@ -48,6 +48,17 @@ class Recordable final : public sdk::trace::Recordable
   void SetDuration(std::chrono::nanoseconds duration) noexcept override;
 
 private:
+  /* Meant for testing purposes */
+  friend class RecordableTestPeer;
+
+  void SetTruncatableString(const int limit,
```

Comment:
Since this function does not need access to any of the class variables, it would be better to only declare it as a local in implementation instead of making it member.